src/yang/ietf-https-notif-transport.yang

```diff
@@ -87,27 +87,7 @@ module ietf-https-notif-transport {
     description
       "A grouping that may be used by other modules wishing to
        configure HTTPS-based notifications without using RFC 8639.";
-    uses httpc:http-client-stack-grouping {
-      refine "transport/tcp" {
-        // create the logical impossibility of enabling the
-        // "tcp" transport (i.e., "HTTP" without the 'S').
-        if-feature "not httpc:tcp-supported";
-      }
-      augment "transport/tls/tls/http-client-parameters" {
-        leaf path {
-          type string;
-          mandatory true;
-          description
-            "A path to the target resources.  Under this
-             path the receiver must support both the 'capabilities'
-             and 'relay-notification' resource targets, as described
-             in RFC XXXX.";
-        }
-        description
-          "Augmentation to add a receiver-specific path for the
-           'capabilities' and 'relay-notification' resources.";
-      }
-    }
+    uses httpc:http-client-grouping;
```

Comment:
I think the `augment "transport/tls/tls/http-client-parameters" {` line needs to be inside this `uses` statement.  

Maybe `augment "/"` or maybe `augment ""` ?

In any casse, the `path` leaf needs to be present...

Reply:
Hmm. Does not like "/", "", or "./". pyang gives similar errors:

`../bin/ietf-https-notif-transport@2024-08-13.yang:91: error: bad value "./" (should be schema-nodeid)`

The http-client-grouping has a `uri` defined in the grouping. Any reason that cannot be used to specify the `path`?

Reply:
`uri` is a `leaf`, so no way to augment a descendent into it!

Ugh, I don't have a good answer...

Maybe try "." for the path?

Reply:
Does not like "." either.

My question was, why can't the `path` be incorporated in the `uri` (not as a descendant node) just as `remote-address` and `remote-port` have been? Demonstrating here with an example.

OLD:
```
<remote-address>receiver.example.com</remote-address>
<remote-port>443</remote-port>
<path>some-path</path>

```
NEW:
`<uri>receiver.example.com:443/some-path</uri>`

Reply:
ah, now I understand your proposal.

I like it, but I think the full fix would require two `uri` leafs, one for the 'capabilities' resource target, and the other for the 'relay-notification' resource target.